### PR TITLE
Updated it_it.json for Splatcraft 3.1.4

### PR DESCRIPTION
**Pull request by Alessio798**
• Translated some new gamerules
• Renamed it-it.json to it_it.json; now it should work fine.

### CODE REVIEW

**Review by Octol1ttle — CHANGES_REQUESTED** [review]
There are missing strings in this translation file:
- Translations for the `/stage` command
- Translations for the arguments of `/stage` command
- Translation for the new Spawn Shield entity
- `item.remote.coords.invalid`

**Comment by Alessio798** [comment]
there should now be everything

File: src/main/resources/assets/splatcraft/lang/it_it.json [hunk 1]
```diff
@@ -409,8 +418,27 @@
...
+  "commands.stage.warp.success": "Teletrasportati %s giocatore/i allo scenario [%s]",
+
   "arg.colorCriterion.notFound": "Nessun criterio geristrato per %s",
   "arg.inkColor.notFound": "Colore sconosciuto: %s",
+  "arg.stageWarp.noSpawnPads": "No available Spawn Pads were found on stage [%s]", # TRANSLATE ME
+  "arg.stageWarp.noPlayers": "No players were found with a Spawn Pad matching their ink color on stage [%s]", # TRANSLATE ME
+  "arg.stage.notFound": "Stage [%s] does not exist", # TRANSLATE ME
+  "arg.stage.alreadyExists": "Stage [%s] already exists", # TRANSLATE ME
+  "arg.stageTeam.notFound": "Unknown team %s for stage [%s]", # TRANSLATE ME
+  "arg.stageSetting.notFound": "Setting %s does not exist", # TRANSLATE ME
```
Comment (Octol1ttle) [inline src/main/resources/assets/splatcraft/lang/it_it.json]: don't forget to remove the `# TRANSLATE ME`s

**Review by Octol1ttle — CHANGES_REQUESTED** [review]

File: src/main/resources/assets/splatcraft/lang/it_it.json (hunk 1 above)
Comment (Alessio798) [reply src/main/resources/assets/splatcraft/lang/it_it.json]: they're now gone.

**Review by Octol1ttle — APPROVED** [review]
Thank you!

**Pull request merged**